src/renderer/components/+catalog/catalog.tsx

```diff
@@ -197,6 +198,85 @@ export class Catalog extends React.Component<Props> {
     );
   }
 
+  renderSingleCategoryList() {
+    return (
+      <ItemListLayout
+        renderHeaderTitle={this.catalogEntityStore.activeCategory?.metadata.name ?? "Browse All"}
+        isClusterScoped
```

Comment:
Since this basically means "load namespace store" I don't think we need `isClusterScoped`.

Reply:
Fixed.

Reply:
Except that we need this... otherwise "namespace selector" is visible 😞

Reply:
That will be fixed by #2743 since it checks the stores (I think)

Reply:
namely:

https://github.com/lensapp/lens/pull/2743/files#diff-e3cb3086ca8de521ccfb9f32c52693b212f25682682504df3d9fcd6515ec7cd1R194-R207